lib/backend/httpbackend/http.go

```diff
@@ -55,9 +56,12 @@ func (f *factory) Create(
 // and http connnection parameters. The URLs come with string format
 // specifiers and define how to pass sha256 parameters
 type Config struct {
-	UploadURL   string `yaml:"upload_url"`   // http upload post url
-	DownloadURL string `yaml:"download_url"` // http download get url
-	Timeout     time.Duration
+	UploadURL       string        `yaml:"upload_url"`   // http upload post url
+	DownloadURL     string        `yaml:"download_url"` // http download get url
+	DownloadTimeout time.Duration `yaml:"download_timeout"`
+
+	// Enables exponential backoff on download network errors.
+	EnableDownloadBackoff bool `yaml:"enable_download_backoff"`
```

Comment:
why not also put the backoff configs here

Reply:
Good point.